query/language/expression.feature

```diff
@@ -456,19 +456,52 @@ Feature: TypeQL Query with Expressions
     Then uniquely identify answer concepts
       | a               | b                |
       | value:integer:1 | value:double:0.5 |
-#    # TODO: 3.x: Re-enable once implemented
-#    When get answers of typeql read query
-#    """
-#      match
-#        let $a = max(2, -3);
-#        let $b = min(2, -3, -5);
-#      select
-#        $a, $b;
-#      """
-#    Then uniquely identify answer concepts
-#      | a               | b                |
-#      | value:integer:2 | value:integer:-5 |
 
+    When get answers of typeql read query
+      """
+      match
+        let $a = max(2, -3);
+        let $b = min(2, -3);
+        let $c = max(10.2dec, 13.5dec);
+        let $d = min(10.2dec, 13.5dec);
+        let $e = max(10.2, 13.5);
+        let $f = min(10.2, 13.5);
+      select
+        $a, $b, $c, $d, $e, $f;
+      """
+    Then uniquely identify answer concepts
+      | a               | b                | c                     | d                     | e                 | f                 |
+      | value:integer:2 | value:integer:-3 | value:decimal:13.5dec | value:decimal:10.2dec | value:double:13.5 | value:double:10.2 |
+
+    Then typeql read query; fails with a message containing: "Built-in expression function 'min' expects '2' arguments but received '0' arguments."
+      """
+      match let $x = min();
+      """
+
+    Then typeql read query; fails with a message containing: "Built-in expression function 'min' expects '2' arguments but received '1' arguments."
+      """
+      match let $x = min(10);
+      """
+
+    Then typeql read query; fails with a message containing: "Built-in expression function 'min' expects '2' arguments but received '3' arguments."
+      """
+      match let $x = min(10, 12, 14);
+      """
+
+    Then typeql read query; fails with a message containing: "Built-in expression function 'max' expects '2' arguments but received '0' arguments."
+      """
+      match let $x = max();
+      """
+
+    Then typeql read query; fails with a message containing: "Built-in expression function 'max' expects '2' arguments but received '1' arguments."
+      """
+      match let $x = max(10);
+      """
+
```

Comment:
Template this maybe? 
I don't think BDD has to be exhaustive, but if we do want it we could make an effort to make the whole thing slightly easier for humans.

We could also test the other error modes - i.e. unsupported pairs of operand types.